continent-countries-1.0-utf8.js

```diff
@@ -5,140 +5,152 @@
  */
 
 /* Dom Ready */
-window.onDomReady = function dgDomReady(fn){
-  if(document.addEventListener)
-    document.addEventListener("DOMContentLoaded", fn, false);
-  else
-    document.onreadystatechange = function(){dgReadyState(fn);}
-}
+window.onDomReady = function dgDomReady(fn) {
+    if (document.addEventListener)
+        document.addEventListener('DOMContentLoaded', fn, false);
+    else
+        document.onreadystatechange = function() {
+            dgReadyState(fn);
+        };
+};
 
-function dgReadyState(fn){
-  if(document.readyState == "interactive") fn();
+function dgReadyState(fn) {
+    if (document.readyState == 'interactive') fn();
 }
 
 /* Object */
 var dgContinentsCountries = function(data) {
-  var defaultData = {
-    continent: false,
-    continentVal: '',
-    country: false,
-    countryVal: '',
-    change: false
-  }
-  for (name in defaultData) {
-    if (!data[name]) {
-      data[name] = defaultData[name];
+    var defaultData = {
+        continent: false,
+        continentVal: '',
+        country: false,
+        countryVal: '',
+        change: false
+    };
+
+    for (name in defaultData) {
```

Comment:
Creating global 'for' variable. Should be 'for (var name ...'.<br>The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.